web/cobrands/fixmystreet/assets.js

```diff
@@ -416,28 +393,62 @@ fixmystreet.add_assets = function(options) {
                 }
                 zoom_to_assets(asset_layer);
             } else {
-                asset_layer.setVisibility(false);
+                if (!asset_layer.fixmystreet.always_visible) {
+                    asset_layer.setVisibility(false);
+                }
                 if (asset_fault_layer) {
                     asset_fault_layer.setVisibility(false);
                 }
             }
         });
-    }
 
-    // Make sure the assets get hidden if the back button is pressed
-    fixmystreet.maps.display_around = (function(original) {
-        function hide_assets() {
-            asset_layer.setVisibility(false);
-            if (asset_fault_layer) {
-                asset_fault_layer.setVisibility(false);
+
+        // Make sure the assets get hidden if the back button is pressed
+        fixmystreet.maps.display_around = (function(original) {
```

Comment:
You only want to do this once now, not on every call to `add()`.

Reply:
🤦‍♂️